coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4353,9 +4353,31 @@ def logical_xor(context, node):
     y = mb.cast(x=y, dtype="bool")
     context.add(mb.logical_xor(x=x, y=y, name=node.name))
 
+
+def _nonzero_as_tuple(context, node, x):
```

Comment:
This util function is great! But it took me some time to understand what it did. It would be really helpful to have a docstring describing what it does (maybe a link to PyTorch's nonzero doc, which illustrates the as_tuple pretty clear).

Reply:
Good idea. I added a docstring.

I decided not to link to PyTorch docs. I'm not a fan of linking to URLs we don't control, since that content can always change.